spec/features/report_multiple_changes_spec.rb

```diff
@@ -95,10 +64,20 @@
     proceed_with "Continue"
 
     expect(page).to have_text "Do you have a letter or paystubs?"
-    choose "No"
+    choose "Yes"
     proceed_with "Continue"
 
-    expect(page).to have_text "Next, tell us about the change in hours or pay."
+    expect(page).to have_text "Add the letter or paystubs."
+    page.attach_file("form[letters][]", Rails.root.join("spec", "fixtures", "image.jpg"), make_visible: true)
```

Comment:
Found a bug with multiple changes and document uploading. We didn't have any others tests around that.